.changelog/2770.txt

```diff
@@ -0,0 +1,8 @@
+```release-note:deprecation
```

Comment:
We should update the release note here to explicitly list all of of the resources marked as deprecated.